qdup.yaml

```
@@ -30,6 +30,7 @@ roles:
       - prepare-images # should be exposed by script files in /modes folder
       - infer-datastouce-hostnames
       - infer-services-hostnames
+      - start-jit-server
```

Ok found this possible issue, shouldn't we start the jit-server **before** building the images? Otherwise during the build the images won't access the JIT server as it is not started up yet

```suggestion
      - start-jit-server
      - prepare-images # should be exposed by script files in /modes folder
      - infer-datastouce-hostnames
      - infer-services-hostnames
```